app/assets/stylesheets/application.css

```diff
@@ -13,6 +13,7 @@
  *= require_tree .
  *= require_self
  *= require foundation_and_overrides
+ *= require top_bar_styling
```

Comment:
This file will get automatically added in via `require_tree`! If you want it to take precedence over Foundation's styling, move line 15 (loading Foundation) to the top so that Foundation loads before anything else!